airbyte-integrations/connectors/source-relational-db/src/main/java/io/airbyte/integrations/source/relationaldb/AbstractDbSource.java

```diff
@@ -252,12 +253,16 @@ private void validateCursorFieldForIncrementalTables(
       }
 
       if (!verifyCursorColumnValues(database, stream.getNamespace(), stream.getName(), cursorField.get())) {
-        tablesWithInvalidCursor.add(
+        tablesWithInvalidCursorToWarnAbout.add(
```

Comment:
You want this to still be `tablesWithInvalidCursor`. We still want to make sure an error is thrown for the Postgres case